Vizzuality/heco-invest
# Fire Google Analytics events for specific actions

This PR adds code to fire some Google Analytics events when specific actions are completed by the user.

The complete list of events can be found on [this spreadsheet](https://docs.google.com/spreadsheets/d/1S7XJbK3FKtmGZNaKVuCcLE2KFY6iJdAhl1_apRdXgeY/edit#gid=0).

## Testing instructions

1. Set this key in your `.env` file:
```
NEXT_PUBLIC_GOOGLE_ANALYTICS=G-33LN47S3NZ
```
2. Install this extension on Chrome: https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna
3. While on localhost:3000, activate the Google Analytics Debugging
![Google Analytics Debugging in on in the extension's menu](https://user-images.githubusercontent.com/6073968/202414322-ffad7c4b-7fdb-4e0d-b03a-810175288ffd.png)
4. Fire each event based on the list of triggers in the [spreadsheet](https://docs.google.com/spreadsheets/d/1S7XJbK3FKtmGZNaKVuCcLE2KFY6iJdAhl1_apRdXgeY/edit#gid=0)

Make sure the events are fired by looking at the console.

## Tracking

[LET-1294](https://vizzuality.atlassian.net/browse/LET-1294).
